Exercises/this/this.js

```diff
@@ -0,0 +1,43 @@
+// Task 1,2
+let student = {
+    name: "Kate",
+    birthYear: 1987,
+    age: function () {
+        const currentYear = new Date().getFullYear()
+        return currentYear - this.birthYear;
+    },
+}
+console.log(student.age());
+const defineAge = student.age;
+console.log(defineAge());
+console.log(defineAge.call(student));
+
+// Task 3
+let graduate = {
+    name: "Alex",
+    birthYear: 1979,
+}
+console.log(defineAge.call(graduate));
+
+// Task 4
+let b = {
+    name: "Peter",
+    greeting: function () {
+        console.log(`Hi, ${this.name}`);
+    }
```

Comment:
Now you use context but not external. `b` object is not a external context for `greeting` function here.